Civi/Core/Container.php

```diff
@@ -160,12 +160,17 @@ public function createContainer() {
 
     $container->setDefinition('psr_log', new Definition('CRM_Core_Error_Log', array()));
 
-    foreach (array('js_strings', 'community_messages') as $cacheName) {
-      $container->setDefinition("cache.{$cacheName}", new Definition(
+    $basicCaches = array(
+      'js_strings' => 'js_strings',
+      'community_messages' => 'community_messages',
+      'checks' => 'checks',
+    );
+    foreach ($basicCaches as $cacheSvc => $cacheGrp) {
```

Comment:
@totten any reason why we are needing the 2 elements rather than just an array like `['js_strings','community_messages','checks']`

Reply:
This is a cherry-pick from #11896, which has a few more draft patches which update the same array. Some of those use-cases could work with `['js_strings','community_messages','checks']`, and one wouldn't. It's easier to juggle the patches (rebasing, re-ordering, cherry-picking, avoiding conflicts, etc) if they all use the use the same multi-line notation.